src/Sulu/Bundle/WebsiteBundle/Analytics/AnalyticsManager.php

```diff
@@ -138,12 +138,12 @@ public function removeMultiple(array $ids)
      * @param string $webspaceKey
      * @param array $data
      */
-    private function setData(AnalyticsInterface $analytics, $webspaceKey, $data)
+    private function setData(AnalyticsInterface $analytics, $webspaceKey, $data): void
     {
-        $analytics->setTitle($this->getValue($data, 'title'));
-        $analytics->setType($this->getValue($data, 'type'));
-        $analytics->setContent($this->getValue($data, 'content', ''));
-        $analytics->setAllDomains($this->getValue($data, 'allDomains', false));
+        $analytics->setTitle($data['title']);
+        $analytics->setType($data['type']);
```

Comment:
To reflect the old behaviour this should be:

```suggestion
        $analytics->setTitle($data['title'] ?? null);
        $analytics->setType($data['type'] ?? null);
```

Reply:
Reverted. We don't change the logic in this PR only the tests.